### DISCUSSION TITLE
Student Developer Pack still to aprove after 5 days

### DISCUSSION THREAD

**Discussion by GuiSantana08** [community/community#142242] [GitHub Education] [answered]
### Hi everyone,

I was approved for the GitHub Student Developer Pack on October 17, but I'm still waiting to gain access to the offers. Should I continue to wait, or could there be an issue with my submission?

Thank you all for your time!

**Comment by davevad93** [comment, marked as answer]
Hi @GuiSantana08 ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @GuiSantana08 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @GuiSantana08 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

**Reply by GuiSantana08** [reply]
Hi @davevad93,

The current status shows: "Congrats! Your request for @GuiSantana08 was approved, and your benefits will be available soon."

I’ll wait a little longer, but if it remains in this state, I’ll raise a ticket as you suggested.

Thanks for your attention!

**Reply by davevad93** [reply]
You're welcome @GuiSantana08 , have a nice day! 👍
